modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/util/KBDropdownItemsProvider.java

```diff
@@ -643,9 +643,7 @@ private UnsafeConsumer<DropdownItem, Exception> _getMoveDropdownItem(
 		KBArticle kbArticle) {
 
 		return dropdownItem -> {
-			dropdownItem.putData("action", "move");
-			dropdownItem.putData(
-				"moveURL",
+			dropdownItem.setHref(
```

Comment:
In my opinion it's more clear to redo the commit story to keep it clean and no not revert things on the same pr.

Reply:
You are right Alicia, however right now redoing the commit story is a bit hard for me. Let's leave it as it is now. Thanks